integration/tests/basic/rpc/test_rpc_neon_estimateScheduledGas.py

```diff
@@ -0,0 +1,307 @@
+        )
+        resp = web3_client_sol.estimate_scheduled(neon_user_no_sols.solana_account.pubkey(), [trx_estimate_obj])
+
+        assert (
+            len(resp["gasList"]) == 1
+        ), f'Amount of transactions must be 1, but actual amount = {len(resp["gasList"])}'
+        assert resp["nonce"] == "0x0"
+        assert resp["maxFeePerGas"] > resp["maxPriorityFeePerGas"], (
+            f"maxFeePerGas must be greater than maxPriorityFeePerGas, "
+            f'but maxFeePerGas = {resp["maxFeePerGas"]} and maxPriorityFeePerGas = {resp["maxPriorityFeePerGas"]}'
+        )
+
+        assert_fields_are_hex(resp, ["chainId", "maxFeePerGas", "maxPriorityFeePerGas", "nonce", "treasuryIndex"])
+
+        assert resp["chainId"] == hex(CHAIN_ID), f'ChainID must be {CHAIN_ID}, but actual = {resp["chainId"]}'
+
+        assert (
+            len(resp["accountList"]) == 6
+        ), f'Amount of accounts must be 6, but actual amount = {len(resp["accountList"])}'
+
+    def test_no_function_in_called_contract(
+        self, web3_client_sol, neon_user, common_contract, evm_loader, treasury_pool
+    ):
+        contract_data = 18
+        data = abi.function_signature_to_4byte_selector("settNumber(uint256)") + eth_abi.encode(
+            ["uint256"], [contract_data]
+        )
+        trx_estimate_obj = ScheduledTrxEstimateRequest(neon_user.checksum_address, common_contract.address, data)
+        resp = web3_client_sol.estimate_scheduled(
+            neon_user.solana_account.pubkey(), [trx_estimate_obj], check_result=False
+        )
+
+        assert "error" in resp, "error field not in response"
+        assert "code" in resp["error"]
+        assert "message" in resp["error"], "message field not in response"
+        assert Error3.CODE == resp["error"]["code"] == 3
+        assert Error3.EXECUTION_REVERTED == resp["error"]["message"]
+
+    def test_wrong_chain_id(self, web3_client_sol, neon_user, common_contract, evm_loader, treasury_pool):
+        contract_data = 18
+        data = abi.function_signature_to_4byte_selector("setNumber(uint256)") + eth_abi.encode(
+            ["uint256"], [contract_data]
+        )
+        trx_estimate_obj = ScheduledTrxEstimateRequest(neon_user.checksum_address, common_contract.address, data)
+
+        transactions = []
+        for trx in [trx_estimate_obj]:
+            trx = {
+                "fromAddress": trx.from_address,
+                "toAddress": trx.to_address,
+                "data": trx.data.hex(),
+                "value": trx.value,
+            }
+            transactions.append(trx)
+        params = {"scheduledSolanaPayer": str(neon_user.solana_account.pubkey()), "transactions": transactions}
+        json = {
+            "jsonrpc": "2.0",
+            "method": "neon_estimateScheduledGas",
+            "params": [params],
+            "id": 0,
+        }
+        resp = requests.post(
+            "http://127.0.0.1:9090/solana/",
```

Comment:
you can't use url like this in tests

Reply:
in this test you can use web3_client instead of web3_client_sol
it will give you other value for chain_id